spiketools/spatial/occupancy.py

```diff
@@ -33,31 +35,51 @@ def compute_spatial_bin_edges(position, bins, area_range=None):
     >>> bins = [5, 4]
     >>> compute_spatial_bin_edges(position, bins)
     (array([1. , 1.8, 2.6, 3.4, 4.2, 5. ]), array([ 6.,  7.,  8.,  9., 10.]))
+
+	Compute bin edges for an example 1d space, with positions 1 through 5.
+    >>> position = np.array([1, 2, 3, 4, 5])
+    >>> bins = [5]
+    >>> compute_spatial_bin_edges(position, bins)
+    array([1. , 1.8, 2.6, 3.4, 4.2, 5. ])
     """
 
-    _, x_edges, y_edges = np.histogram2d(position[0, :], position[1, :],
+    # 2d case
+    if (len(bins) == 2):
```

Comment:
What about testing `if position.ndim == 2`? That seems like a slightly more direct test than inferring the size from the bin definition.